Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -394,7 +360,7 @@ function Invoke-CodeGen {
         }
 
         $parameters = @(
-            "-r", $RepositoryRoot.ToLower(),
+            "-r", $RepositoryRoot,
```

Comment:
This file isn't completely migrated yet, [ODS-5378](https://tracker.ed-fi.org/browse/ODS-5378) will complete its migration.
I've removed the `ToLower` so that we can at least run initdev in Linux and ease testing, note that `ToLower` isn't needed in Windows either.